### ISSUE TITLE
Bug: Exporting Resultset to new table in Oracle

### ISSUE THREAD

**Issue by finlbi** [dbeaver/dbeaver#3183]
When attempting to export data via the Oracle driver, the following error is generated when any fields with the number datatype are in the resultset.

```
CREATE TABLE TEST_TABLE(
FIELD1 NUMBER(39,0),
FIELD2 VARCHAR2(50) NOT NULL,
FIELD3 VARCHAR2(7));
```

> Error starting at line : 2 in command -
> Error report -
> ORA-01727: numeric precision specifier is out of range (1 to 38)
> 01727. 00000 -  "numeric precision specifier is out of range (1 to 38)"
> *Cause:    
> *Action:

If the table creation query is manually modified and run using a value within the range specified, the table is created successfully.

```
CREATE TABLE TEST_TABLE(
FIELD1 NUMBER(38,0),
FIELD2 VARCHAR2(50) NOT NULL,
FIELD3 VARCHAR2(7));
```

> Table TEST_TABLE created.

**Comment by serge-rider** [comment]
You can change target DDL manually (click on Column Mappings button and then change target data type by clicking on column type name)

**Comment by finlbi** [comment]
Awesome, thanks for the tip!

Is there a way to have the DDL default to NUMBER(38, 0) instead of NUMBER(39, 0) instead of having to manually modify the DDL each time?

According to [their documentation](https://docs.oracle.com/cd/B28359_01/server.111/b28318/datatype.htm#CNCPT313), the number data type supports up to 38 significant digits.

**Comment by finlbi** [comment]
The bug fixes and improvements in v5.0.6 seem to have resolved this issue. Thank you!
